packages/dom/src/dom/clean-node-list.js

```diff
@@ -11,27 +11,44 @@ import remove from './remove';
 import unwrap from './unwrap';
 import { isPhrasingContent } from '../phrasing-content';
 import insertAfter from './insert-after';
+import isElement from './is-element';
+
+/* eslint-disable jsdoc/valid-types */
+/**
+ * @typedef SchemaItem
+ * @property {string[]} [attributes] Attributes.
+ * @property {(string | RegExp)[]} [classes] Classnames or RegExp to test against.
+ * @property {'*' | { [tag: string]: SchemaItem }} [children] Child schemas.
+ * @property {unknown[]} [require] ???
+ * @property {boolean} allowEmpty Whether to allow empty ???.
```

Comment:
I'm not sure how to describe these properties. Any ideas @ellatrix?

Reply:
I think whether to allow the element to be empty